devel/spicy/tutorial.rst

```diff
@@ -208,8 +208,16 @@ function that Spicy makes available once we have successfully parsed a
 request: :spicylink:`spicy::accept_input
 <programming/library.html#spicy-accept-input>`. That function signals
 the host application---i.e., Zeek in our case—--that the parser is
-processing the expected protocol. With that, our request looks like
-this now:
+processing the expected protocol.
+
+First, we need to make sure the library function is imported in
```

Comment:
```suggestion
First, we need to make sure the Spicy standard library is imported, so that we will have its functions available:
```

Reply:
Sure, I did keep the filename referenced though